pkgs/applications/audio/noson/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, flac
+, libpulseaudio
+, qtbase
+, qtgraphicaleffects
+, qtquickcontrols2
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
I think we should use the `finalAttrs pattern` here? See: https://github.com/NixOS/nixpkgs/pull/119942

Reply:
Oh that's cool! Thanks for pointing it out; fixed and force-pushed; comparison at https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe